test/stdlib/Inputs/SwiftObjectNSObject/SwiftObjectNSObject.m

```diff
@@ -159,12 +162,24 @@ void TestSwiftObjectNSObject(id c, id d)
   expectFalse([C_meta isEqual:D_meta]);
   expectFalse([S_meta isEqual:C_meta]);
 
+  // Check that ObjC isEqual delegates to Equatable
+  expectTrue([e1a isEqual: e1b]);
+  expectFalse([e1a isEqual: e2]);
+  expectFalse([e1b isEqual: e2]);
```

Comment:
```suggestion
  expectTrue([e1a isEqual: e1a]);
  expectFalse([e1a isEqual: e1b]);
  expectFalse([e1a isEqual: e2]);
  expectFalse([e1b isEqual: e2]);
```

Reply:
See above.  I now use a constant for the hashValue of non-Hashable types, which I believe means that we can continue to bridge Swift `==` in these cases.